Examples/BookingSystem.AspNetCore/Stores/FacilityStore.cs

```diff
@@ -555,13 +569,27 @@ protected override async ValueTask ProposeOrderItems(List<OrderItemContext<Facil
                 var (result, bookedOrderItemInfos) = await FakeDatabase.BookOrderItemsForFacilitySlot(
                     databaseTransaction.FakeDatabaseTransaction,
                     flowContext.OrderId.ClientId,
-                    flowContext.SellerId.SellerIdLong ?? null /* Hack to allow this to work in Single Seller mode too */,
+                    flowContext.SellerId.IdLong ?? null /* Hack to allow this to work in Single Seller mode too */,
                     flowContext.OrderId.uuid,
                     ctxGroup.Key.SlotId.Value,
                     RenderOpportunityId(ctxGroup.Key),
                     RenderOfferId(ctxGroup.Key),
                     ctxGroup.Count(),
-                    true
+                    true,
+                    ctxGroup
+                        .Where(x => x.RequestOrderItem.Attendee != null)
+                        .Select(x => new FakeDbPerson
+                        {
+                            GivenName = x.RequestOrderItem.Attendee.GivenName,
+                            FamilyName = x.RequestOrderItem.Attendee.FamilyName,
+                            Email = x.RequestOrderItem.Attendee.Email,
+                            Telephone = x.RequestOrderItem.Attendee.Telephone,
+                        })
+                        .ToList(),
```

Comment:
@civsiv will this not cause issues with the attendees not being stored against the correct OrderItems if only some attendees are specified.

e.g. at P

1: Bob
2: Null
3: Alex

then B would be

1: Bob
2: Alex
3: Null

So we'd actually want something more like the below:?

```suggestion
                        .Select(x => x.RequestOrderItem.Attendee == null ? null : new FakeDbPerson
                        {
                            GivenName = x.RequestOrderItem.Attendee.GivenName,
                            FamilyName = x.RequestOrderItem.Attendee.FamilyName,
                            Email = x.RequestOrderItem.Attendee.Email,
                            Telephone = x.RequestOrderItem.Attendee.Telephone,
                        })
                        .ToList(),
```